cylc/uiserver/jupyterhub_config.py

```diff
@@ -49,21 +55,35 @@ def _load(path):
         exec(path.read_text())
 
 
+def get_conf_dir_hierarchy(config_paths: List[Path]):
+    """Takes list of config paths, adds version and filename to the path"""
+    conf_hierarchy = []
+    version_hierarchy = get_version_hierarchy(hub_version or __version__)
+    for x in product(config_paths, version_hierarchy):
+        conf_hierarchy.append(Path(x[0], x[1], 'jupyter_config.py'))
+    return conf_hierarchy
+
+
 def load():
```

Comment:
(Type annotations in a function body aren't checked unless the definition has type annotations)

```suggestion
def load() -> None:
```